src/config/localization/translations.json

```diff
@@ -484,23 +263,6 @@
   "Price per %symbol% with fee:": "Price per %symbol% with fee:",
   "You need an active PancakeSwap Profile to take part in an IFO!": "You need an active PancakeSwap Profile to take part in an IFO!",
   "Achievement": "Achievement",
-  "Commit ~%minLp% LP in total to earn!": "Commit ~%minLp% LP in total to earn!",
-  "What’s the difference between a Basic Sale and Unlimited Sale?": "What’s the difference between a Basic Sale and Unlimited Sale?",
-  "In the Basic Sale, every user can commit a maximum of about 100 USD worth of CAKE. We calculate the maximum CAKE amount about 30 minutes before each IFO. The Basic Sale has no participation fee.": "In the Basic Sale, every user can commit a maximum of about 100 USD worth of CAKE. We calculate the maximum CAKE amount about 30 minutes before each IFO. The Basic Sale has no participation fee.",
-  "In the Unlimited Sale, there’s no limit to the amount of CAKE you can commit. However, there’s a fee for participation: see below.": "In the Unlimited Sale, there’s no limit to the amount of CAKE you can commit. However, there’s a fee for participation: see below.",
-  "Which sale should I commit to? Can I do both?": "Which sale should I commit to? Can I do both?",
-  "You can choose one or both at the same time! If you’re only committing a small amount, we recommend the Basic Sale first. Just remember you need a PancakeSwap Profile in order to participate.": "You can choose one or both at the same time! If you’re only committing a small amount, we recommend the Basic Sale first. Just remember you need a PancakeSwap Profile in order to participate.",
-  "How much is the participation fee?": "How much is the participation fee?",
-  "There’s only a participation fee for the Unlimited Sale: there’s no fee for the Basic Sale.": "There’s only a participation fee for the Unlimited Sale: there’s no fee for the Basic Sale.",
-  "The fee will start at 1%.": "The fee will start at 1%.",
-  "The 1% participation fee decreases in cliffs, based on the percentage of overflow from the “Unlimited” portion of the sale.": "The 1% participation fee decreases in cliffs, based on the percentage of overflow from the “Unlimited” portion of the sale.",
-  "Where does the participation fee go?": "Where does the participation fee go?",
-  "The CAKE from the participation fee will be thrown into the weekly token burn.": "The CAKE from the participation fee will be thrown into the weekly token burn.",
```

Comment:
used in `src/views/Ifos/components/IfoQuestions/config.ts`

Reply:
Why it is removed?

Reply:
no, it's what I found unused by checking `t(..)`, since `IfoQuestions/config.ts` are only string. 
I'm figuring out another way to find unused translation

Reply:
Ok, we should convert that constant to function and pass t as parameter then it will be checked in unit test. Should we do it as a seperate pr?

Reply:
Like here
https://github.com/pancakeswap/pancake-frontend/blob/develop/src/views/Nft/market/Home/config.ts

Reply:
I think it becomes verbose and restrict to pass t in everywhere..
I was checking if there's any good node js pkg like `grep`, we only have to check if those text is inside our code

Reply:
Check my pr: 

https://github.com/pancakeswap/pancake-frontend/pull/2773

Reply:
> 

Then we will need to define rules to understand that text is to be translated or not. I was checking this when I was writing that translation unit tests but couldn't find proper one